### ISSUE TITLE
FileManager: right clicking an item the second time should work

### ISSUE THREAD

**Issue by diversys** [SerenityOS/serenity#4649]
* Right click a link on the Desktop
* Right click another link
* Nothing happens

**Comment by linusg** [comment]
Working fine here:

![Peek 2020-12-30 19-36](https://user-images.githubusercontent.com/19366641/103373850-584f9780-4ad6-11eb-9741-6259cd1ef36c.gif)

**Comment by diversys** [comment]
Yes, because you close context menus in-between right clicks. The idea is that you don't have to do that to open subsequent context menus.

**Comment by linusg** [comment]
> The idea is that you don't have to do that to open subsequent context menus.

That wasn't clear to me at all from this very short issue description :)

> Nothing happens

That's not true: the open context menu closes, if any, or a new one is opened as one would expect. Once again this matches what GNOME does, I can't test a new or old Windows at the moment, but this seems like sensible behaviour to me.

**Comment by diversys** [comment]
At east on Windows, macOS and Haiku it works as expected.

**Comment by X547** [comment]
> Once again this matches what GNOME does

GNOME 3+ is really bad reference. Most behavior (including menu opening) is the same in Windows, Mac OS, Haiku, Qt, but different in GNOME 3+.

**Comment by linusg** [comment]
someone go ahead and change it then :)
